docs/content/python/new-roles/getting-started.md

```diff
@@ -92,6 +92,23 @@ class as well as a SQLAlchemy `sessionmaker` to the
     hlOpts="hl_lines=6"
 >}}
 
+Oso stores role and permission configuration in your database alongside
```

Comment:
I moved this up here, since there's no longer any need to go through the setup steps lower down.

Reply:
I don't think this is correct. `enable_roles` doesn't delete and replace all configuration, `synchronize_data` does.